File: app/src/main/java/com/kenvix/newsproject/newsproject/android/ui/main/AboutFragment.java
package com.kenvix.newsproject.newsproject.android.ui.main;

import android.view.View;
import android.widget.TextView;

import com.kenvix.newsproject.newsproject.android.ui.base.BaseFragment;
import com.kenvix.utils.android.annotation.ViewAutoLoad;

public class AboutFragment extends BaseFragment {
    //@ViewAutoLoad
    public TextView mainAboutMessage;

    @Override
    protected void onInitialize(View view) {

    }

    @Override
    protected int getFragmentContentLayout() {
        return 0;
    }

    @Override
    protected int getBaseActivityContainer() {
        return 0;
    }
}

File: app/src/main/java/com/kenvix/walk/ui/main/AboutFragment.kt
package com.kenvix.walk.ui.main

import android.view.View
import android.widget.TextView

import com.kenvix.walk.ui.base.BaseFragment

internal class AboutFragment : BaseFragment() {
    //@ViewAutoLoad
    var mainAboutMessage: TextView? = null

    override fun onInitialize(view: View) {

    }

    override fun getFragmentContentLayout(): Int {
        return 0
    }

    override fun getBaseActivityContainer(): Int {
        return 0
    }
}
